integration/flink/shared/src/test/resources/events/expected_cassandra.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Looks like those files were copied, not moved - is that intentional?

Reply:
It is not intentional. I moved whole resources folder. Only json files lose the git history. Do you have any suggestion on this?

Reply:
@mobuchowski 
Double checked in IntelliJ. The git history is kept for these files, as git blame still shows original author. I guess it is a bug in GitHub for this json files.